### PR TITLE
Add `rollbar.warning` and change CI to use `yarn`

### PR DESCRIPTION
**Pull request by nf-s**
- Add `rollbar.warning` for `TerriaErrorSeverity.Warning`
* Use `yarn` in CI scripts (and upgrade node to v14)
* change `npm run` usage to `yarn` 

TerriaMap PR to change `npm run` usage to `yarn` - https://github.com/TerriaJS/TerriaMap/pull/560

### Checklist

-   [ ] ~There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)~
-   [ ] ~I've updated relevant documentation in `doc/`.~
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

File: lib/Models/ErrorServiceProviders/RollbarErrorServiceProvider.ts [hunk 1]
```diff
@@ -21,6 +21,12 @@ export default class RollbarErrorServiceProvider
   }
 
   error(error: string | Error | TerriaError) {
-    this.rollbar.error(error instanceof TerriaError ? error.toError() : error);
+    if (error instanceof TerriaError) {
+      error.severity === TerriaErrorSeverity.Error
```
Comment (na9da) [inline lib/Models/ErrorServiceProviders/RollbarErrorServiceProvider.ts]: The type declaration says `error.severity` be a `() => TerriaErrorSeverity`. Should we handle that as well?

Comment (nf-s) [reply lib/Models/ErrorServiceProviders/RollbarErrorServiceProvider.ts]: oh yeah - nice pickup - sorry forgot about that

Comment (nf-s) [reply lib/Models/ErrorServiceProviders/RollbarErrorServiceProvider.ts]: donee

File: doc/getting-started.md [hunk 2]
```diff
@@ -15,7 +15,9 @@ cd TerriaMap
 
 export NODE_OPTIONS=--max_old_space_size=4096
 
-npm install && npm run gulp && npm start
+npm install -g yarn
+
+yarn install && npm run gulp && npm start
```
Comment (wibily) [inline doc/getting-started.md]: yarn gulp && yarn start?

File: buildprocess/ci-deploy.sh [hunk 3]
```diff
@@ -29,9 +30,9 @@ git config --global user.email "info@terria.io"
 git config --global user.name "GitHub Actions"
 git commit -a -m 'temporary commit' # so the version doesn't indicate local modifications
 git tag -a "TerriaMap-$TERRIAMAP_COMMIT_HASH--TerriaJS-$TERRIAJS_COMMIT_HASH" -m 'temporary tag'
-rm package-lock.json # because TerriaMap's package-lock.json won't reflect terriajs dependencies
-npm install
-npm install moment@2.24.0
+rm yarn.lock # because TerriaMap's yarn.lock won't reflect terriajs dependencies
+yarn install
+yarn add -W moment@2.24.0
 npm run gulp build
```
Comment (wibily) [inline buildprocess/ci-deploy.sh]: yarn gulp build

File: doc/getting-started.md (hunk 2 above)
Comment (nf-s) [reply doc/getting-started.md]: done

File: buildprocess/ci-deploy.sh (hunk 3 above)
Comment (nf-s) [reply buildprocess/ci-deploy.sh]: done

**Review by wibily — APPROVED** [review]
thanks, this was blocking the build for the other PR

**Pull request merged**